v19.2/use-the-built-in-sql-client.md

```diff
@@ -740,6 +741,63 @@ Time: 2.426534ms
 > SHOW DATABASE
 ~~~
 
+### Repeat a SQL statement
+
+Use the `--execute` flag to create a table from the command line:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure \
+--execute="CREATE TABLE t1 (id UUID DEFAULT gen_random_uuid(), name STRING)"
+~~~
+
+With the `--watch` flag, you can repeat the statements specified with the `--execute` flag periodically, until a SQL error occurs or the process is terminated:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure \
+--execute="INSERT INTO t1 (name) VALUES ('a'), ('b'), ('c')" \
+--watch 3s
+~~~
+
+In this example we kill the process with Ctrl+C.
```

Comment:
not a great example. Applications that perform INSERTs want to insert different data every time, and that's not going to work.

Also the feature is really intended for _monitoring_ so it would be interesting to have a monitoring-oriented example.

For example perhaps something like `--watch 1h -e 'select count(*)'` to monitor table size, or `--watch 1s -e 'select sum(count) from crdb_internal.node_statement_statistics'` for the number of queries run on a given node.